internal/stack/_static/docker-compose-stack.yml.tmpl

```
@@ -170,11 +170,12 @@ services:
       interval: 60s
       timeout: 50s
       retries: 5
-    command: bash -c 'if [[ ! $(bin/logstash-plugin list) == *"logstash-filter-elastic_integration"* ]]; then echo "Missing plugin logstash-filter-elastic_integration, installing now" && bin/logstash-plugin install logstash-filter-elastic_integration; fi && bin/logstash -f /usr/share/logstash/pipeline/logstash.conf'
+    command: bash -c 'chmod +x /usr/share/logstash/startup.sh && /usr/share/logstash/startup.sh'
     volumes:
       - "../certs/logstash:/usr/share/logstash/config/certs"
       - "../certs/elasticsearch/cert.pem:/usr/share/logstash/config/certs/elasticsearch.pem"
```

To ensure that we neither try to write here when running locally.
```suggestion
      - "../certs/logstash:/usr/share/logstash/config/certs:ro"
      - "../certs/elasticsearch/cert.pem:/usr/share/logstash/config/certs/elasticsearch.pem:ro"
```

---

It seems making `config/certs` read-only, mounting `elasticsearch.pem` is failing?! Removing read-only from certs works fine, we need to mount file one by one if it is strictly required.
```
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/opt/buildkite-agent/.elastic-package/profiles/logstash/certs/elasticsearch/cert.pem" to rootfs at "/usr/share/logstash/config/certs/elasticsearch.pem": open /var/lib/docker/overlay2/c73fe26af0e42262b9972a5dc32a073d3141e9dee2467ca8ed3bc5d07ce6733b/merged/usr/share/logstash/config/certs/elasticsearch.pem: read-only file system: unknown
```